src/otelcollector/otelcol-config.yml

```diff
@@ -9,6 +9,9 @@ exporters:
     tls:
       insecure: true
   logging:
+  prometheus:
+    endpoint: 0.0.0.0:8889
```

Comment:
Maybe use the default port? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#prometheus-exporter

Reply:
Done, I set endpoint to 0.0.0.0:9464 now, since in the configuration of prometheus exporter, the endpoint info must be specified, otherwise it will not start